vim/vscode-neovim.vim

```diff
@@ -157,7 +97,9 @@ augroup VscodeGeneral
     " Looks like external windows are coming with "set wrap" set automatically, disable them
     " autocmd WinNew,WinEnter * :set nowrap
     autocmd WinScrolled * call VSCodeExtensionNotify('window-scroll', win_getid(), winsaveview())
-    autocmd ModeChanged * call VSCodeExtensionNotify('mode-changed', v:event.old_mode, v:event.new_mode)
+    autocmd ModeChanged * call VSCodeExtensionNotify('mode-changed', v:event.new_mode)
+    " LazyVim will clear runtimepath by default. To avoid user intervention, we need to set it again.
+    autocmd User LazyDone let &runtimepath = &runtimepath . ',' . s:luaPath
```

Comment:
😬

Reply:
Any better suggestions? The other option would be to ask lazyvim author to detect `g:vscode` and not clear rtp. Either way, there needs to be some custom logic to allow custom lua files.